### ISSUE TITLE
pdf version of the cheatsheet is not created anymore

### ISSUE THREAD

**Issue by SimeonEhrig** [alpaka-group/alpaka#2038]
In the documentation we have a [alpaka cheatsheet](https://alpaka.readthedocs.io/en/latest/basic/cheatsheet.html). At the moment only the HTML version is working. It is not possible to download the pdf version anymore.

**Comment by SimeonEhrig** [comment]
I set up a test readthedocs page of my alpaka fork to test bug fixes because the preview build in pull request does not build the pdf version and because of it it does not fail. The test instance simply works. I use the current version of the develop branch and it simply builds.

I found only one difference. In the original alpaka readthedocs it installs some dependencies automatically, like `pillow` and `alabaster`. This was default behavoir and will not be changed, if you create the readthedoc site before the 7.8.23. Maybe this causes the problem. No idea how to disable this automatically installed dependencies.

https://docs.readthedocs.io/en/stable/build-default-versions.html#python
